src/windows/Program.cs

```diff
@@ -4,19 +4,56 @@
 using Microsoft.Diagnostics.Tracing.Parsers;
 using Microsoft.Diagnostics.Tracing.Analysis;
 using Microsoft.Diagnostics.Tracing.Analysis.GC;
-using System.Collections.Generic;
 using System.Threading;
+using CommandLine;
+using System.Diagnostics;
 
 namespace realmon
 {
     class Program
     {
+        public class Options
+        {
+            [Option(shortName: 'n',
+                    longName: "processName",
+                    Required = false,
+                    HelpText = "The process name for which the GC Monitoring will take place for - the first process is chosen if there are multiple.")]
+            public string ProcessName { get; set; } = null;
+
+            [Option(shortName: 'p',
+                    longName: "processId",
+                    Required = false,
+                    HelpText = "The process id for which the GC Monitoring will take place for.")]
+            public int ProcessId { get; set; } = -1;
+
+            [Option(shortName: 'm',
+                    longName: "minDurationForGCPauseMs",
```

Comment:
nit - instead of `minDurationForGCPauseMs` could we use `minDurationForGCPauseMSec` just to follow our tooling code convention :)